### DISCUSSION TITLE
Is it possible to jump to an Obsidian note from a reference in Zotero?

### DISCUSSION THREAD

**Discussion by fangbq** [retorquere/zotero-better-bibtex#2037] [Q&A]
We can create and open an obsidian note in Obsidian for any imported references from Zotero now.

What if I want to open the corresponding obsidian note when I am reading a pdf in Zotero (using the internal Zotero pdf viewer)?
Is there any way to jump/link to Obsidian from Zotero?

**Comment by glocalglocal** [comment]
Zotero Obsidian Citations does that and more. https://github.com/daeh/zotero-obsidian-citations
